src/pages/HomePage/HomePage.js

```diff
@@ -1,7 +1,9 @@
-// eslint-disable-next-line no-unused-vars
```

Comment:
Can you leave these eslint comments in? This just helps stop our code from throwing console warnings since it doesn't yet use the 'user' Context variable